File: web/js/shinow/app/model/DeliveryInfo.js
/**
 * <p>
 * 配送商信息表
 * </p>
 * 
 * @author 代码生成器
 * @class app.model.DeliveryInfo
 * @extends Ext.data.Model
 */
Ext.define('app.model.DeliveryInfo', {
		extend : 'Ext.data.Model',
		alias : 'model.DeliveryInfo',
		fields : [
				{
					name : 'deliveryCode',
					type: 'string'
				},
				{
					name : 'deliveryName',
					type: 'string'
				},
				{
					name : 'address',
					type: 'string'
				},
				{
					name : 'linkName',
					type: 'string'
				},
				{
					name : 'linkTel',
					type: 'string'
				},
				{
					name : 'qq',
					type: 'string'
				},
				{
					name : 'email',
					type: 'string'
				},
				{
					name : 'sortId',
					type: 'int'
				},
				{
					name : 'status',
					type: 'string'
				}
		]
});

File: src/main/java/com/shinowit/entity/DeliveryInfo.java
package com.shinowit.entity;

import java.io.Serializable;

public class DeliveryInfo implements Serializable {
    private String deliveryCode;

    private String deliveryName;

    private String address;

    private String linkName;

    private String linkTel;

    private String qq;

    private String email;

    private Byte sortId;

    private Boolean status;

    private static final long serialVersionUID = 1L;

    public String getDeliveryCode() {
        return deliveryCode;
    }

    public void setDeliveryCode(String deliveryCode) {
        this.deliveryCode = deliveryCode;
    }

    public String getDeliveryName() {
        return deliveryName;
    }

    public void setDeliveryName(String deliveryName) {
        this.deliveryName = deliveryName;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getLinkName() {
        return linkName;
    }

    public void setLinkName(String linkName) {
        this.linkName = linkName;
    }

    public String getLinkTel() {
        return linkTel;
    }

    public void setLinkTel(String linkTel) {
        this.linkTel = linkTel;
    }

    public String getQq() {
        return qq;
    }

    public void setQq(String qq) {
        this.qq = qq;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public Byte getSortId() {
        return sortId;
    }

    public void setSortId(Byte sortId) {
        this.sortId = sortId;
    }

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(Boolean status) {
        this.status = status;
    }
}